File: D4457_LJ01-ADD-JKT/Problem_B_TraditionalMarket.cpp
#include <stdio.h>

int main(){
    long int d,s,t;
    

    scanf("%ld %ld %ld", &d, &s, &t);

    if (s>t && t>d){
        printf("Sayur\n");
        printf("Telur\n");
        printf("Daging\n");
    }

	else if (s>d && t<d){
        printf("Sayur\n");
        printf("Daging\n");
        printf("Telur\n");
    }
    
    else if (d>t && t>s){
        printf("Daging\n");
        printf("Telur\n");
        printf("Sayur\n");
    }
    
	else if (d>s && t<s){
        printf("Daging\n");
        printf("Sayur\n");
        printf("Telur\n");
    }

	else if (t>d && d>s){
        printf("Telur\n");
        printf("Daging\n");
        printf("Sayur\n");
    }

   else if (t>s && d<s){
        printf("Telur\n");
        printf("Sayur\n");
        printf("Daging\n");
    }

	return 0;
 }

File: D4457_LJ01-ADD-JKT/Problem_B_TraditionalMarket.py
inputNilai=input().split(" ");

harga=[];
for i in range(len(inputNilai)):
    harga.append(int(inputNilai));

daging=harga[0];
sayur=harga[1];
telor=harga[2];
harga.sort(reverse=true);

for i in range(len(harga)):
    if harga[i]==daging:
        print("daging");
    elif harga[i]==sayur:
        print("sayur");
    elif harga[i]==telor:
        print("telur");